src/pages/Post/Post.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable prettier/prettier */
+import './Post.css'
+
+import axios from 'axios'
+import React, {useEffect, useState } from 'react'
+
+interface IPost {
+    id: number,
+    links: {id: number, caption: string, url: string}[],
+    caption: string, 
+    short_text: string, 
+    details: string,
+    added_at: string,
+    show_after: string,
+    disable_after: string
+}
+
+const defaultPosts: IPost[] = []

```

Comment:
budu nejake defaultne takto oddelene? cakal by som, ze to bude vzdy prazdne pole, tak by som nerobil osobitnu konstantu... tym padom by som zmenil ponukany `useState(defaultPosts)` na `useState<IPost[]>([])`, lebo typescript nevie, akeho typu je pole `[]`, pripadne `useState([] as IPost[])` by malo byt tiez validne riesenie